packages/atomic/src/components/search/facets/atomic-rating-range-facet/atomic-rating-range-facet.tsx

```diff
@@ -344,7 +353,7 @@ export class AtomicRatingRangeFacet
   }
 
   private get valuesToRender() {
-    return this.facetState.values.filter(
+    return this.facet.state.values.filter(
```

Comment:
Helps with race condition between update of facetState object and the isHidden() call.